### PR TITLE
Add teleport to player console command

### PR DESCRIPTION
**Pull request by zionnBE**
Teleports the current player to another player's location when given their name.

### CODE REVIEW

File: Robust.Server/Console/Commands/PlayerCommands.cs [hunk 1]
```diff
@@ -61,6 +62,38 @@ public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
...
+            {
+                if (target.AttachedEntity == null)
+                    return;
+
+                var posX = target.AttachedEntity.Transform.Coordinates.X;
+                var posY = target.AttachedEntity.Transform.Coordinates.Y;
+                var mapId = target.AttachedEntity.Transform.Coordinates.GetMapId(IoCManager.Resolve<IEntityManager>())
+                    .ToString();
+
+                shell.ExecuteCommand(player, $"tp {posX} {posY} {mapId}");
```
Comment (gradientvera) [inline Robust.Server/Console/Commands/PlayerCommands.cs]: Just set the player's AttachedEntity's coordinates to the target's AttachedEntity's coordinates directly.

**Review by DrSmugleaf — APPROVED** [review]

File: Robust.Server/Console/Commands/PlayerCommands.cs [hunk 2]
```diff
@@ -61,6 +62,38 @@ public void Execute(IConsoleShell shell, IPlayerSession? player, string[] args)
         }
     }
 
+    public class TeleportToPlayerCommand : IClientCommand
+    {
+        public string Command => "tpto";
+        public string Description => "Teleports a player to the location of another player.";
```
Comment (gradientvera) [inline Robust.Server/Console/Commands/PlayerCommands.cs]: This description makes it seem like you can teleport any player to any other player.

**Review by gradientvera — CHANGES_REQUESTED** [review]

**Comment by gradientvera** [comment]
Good first PR, btw. Thanks!

File: Robust.Server/Console/Commands/PlayerCommands.cs (hunk 2 above)
Comment (zionnBE) [reply Robust.Server/Console/Commands/PlayerCommands.cs]: Fixed.

File: Robust.Server/Console/Commands/PlayerCommands.cs (hunk 1 above)
Comment (zionnBE) [reply Robust.Server/Console/Commands/PlayerCommands.cs]: Fixed.

**Review by gradientvera — APPROVED** [review]

**Comment by KingTerrytheTerrible** [comment]
fuck you zionn you fucking idiot

**Comment by ghost** [comment]
:V

**Comment by gradientvera** [comment]
let's not

**Pull request merged**